test/e2e/storage/volume_provisioning.go

```diff
@@ -103,7 +103,7 @@ func testDynamicProvisioning(t storageClassTest, client clientset.Interface, cla
 	// Check sizes
 	expectedCapacity := resource.MustParse(t.expectedSize)
 	pvCapacity := pv.Spec.Capacity[v1.ResourceName(v1.ResourceStorage)]
-	Expect(pvCapacity.Value()).To(Equal(expectedCapacity.Value()), "pvCapacity is not equal to expectedCapacity")
+	Expect(pvCapacity.Value()).To(Equal(expectedCapacity.Value()), "pvCapacity %b is not equal to expectedCapacity %b", pvCapacity.Value(), expectedCapacity.Value())
```

Comment:
The Expect() function should actually print out the values

Reply:
I think you can skip this change.  The Expect() function should already print out the two values.